bin/shuji.js

```diff
@@ -163,28 +163,24 @@ fileList.forEach((filepath) => {
   }
 
   const input = fs.readFileSync(filepath, 'utf8'),
-    outdir = path.join(outputDir, path.dirname(filepath)),
     output = shuji(input, {
       verbose: typeof opts.verbose === 'boolean' ?
         opts.verbose :
         false
     });
 
-  fs.ensureDirSync(outdir);
-
   Object.keys(output).forEach((item) => {
-    const outfile = path.join(outdir, item);
+    // https://security.stackexchange.com/a/123723
+    const safeSuffix = path.normalize(item).replace(/^(\.\.[/\\])+/, '');
+    const outfile = path.resolve(outputDir, safeSuffix);
+
+    fs.ensureDirSync(path.dirname(outfile));
 
     if (opts.verbose) {
       console.log(`Writing to file ${outfile}`);
     }
 
-    if (fs.existsSync(outfile)) {
```

Comment:
Is there a reason why there should not be a complaiment when the given file already exists?

Reply:
What would you like to see here?

I just tried this PR out and iiiiiiiit's gRRRReat !  Would love to see it merged pending this change

Reply:
@paazmaya maybe adding new `--force` option to overwrite existing files?

Reply:
`--force` or `--overwrite` would be great